### PR TITLE
Chore: Added menu page e2e tests (Issue/3468)

### PR DESCRIPTION
**Pull request by lemmyadams**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
addresses adaptlearning/adapt-contrib-boxMenu#167

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]

**Review by eleanor-heath — APPROVED** [review]

File: test/e2e/menuPage.cy.js [hunk 1]
```diff
@@ -1,8 +1,42 @@
...
+    cy.get('.menu-item').each(($item, index) => {
+      cy.get($item).within(() => {
+        cy.get('.menu-item__title').should('contain', Content[index].displayTitle)
+        cy.get('.menu-item__body').should('contain', Content[index].body)
+        cy.get('button').should('contain', Content[index].linkText)
+        cy.get('.menu-item__duration').should('contain', Content[index].duration)
+        cy.get('img.menu-item__image').should('exist').should('have.attr', 'src', Content[index]._graphic.src)
+      })
+    })
+  })
```
Comment (oliverfoster) [inline test/e2e/menuPage.cy.js:26]: This should really live in adapt-contrib-boxmenu

File: test/e2e/menuPage.cy.js [hunk 2]
```diff
@@ -1,8 +1,42 @@
...
+    cy.get('button[data-event="toggleDrawer"]').click()
+
+    cy.get('.drawer[aria-expanded="true"]').should('exist')
+    cy.get('.drawer[aria-expanded="false"]').should('not.exist')
+
+    cy.get('button.drawer__close-btn').click()
+
+    cy.get('.drawer[aria-expanded="true"]').should('not.exist')
+    cy.get('.drawer[aria-expanded="false"]').should('exist')
+  })
```
Comment (oliverfoster) [inline test/e2e/menuPage.cy.js:41]: This should be in a drawer.cy.js file

File: test/e2e/menuPage.cy.js (hunk 1 above)
Comment (oliverfoster) [reply test/e2e/menuPage.cy.js]: I mean that it should live in the plugin adapt-contrib-boxmenu

**Comment by joe-allen-89** [comment]
New data loading needs to be implemented.

**Comment by cahirodoherty-learningpool** [comment]
Replaced with https://github.com/adaptlearning/adapt-contrib-boxMenu/pull/168 and https://github.com/adaptlearning/adapt-contrib-resources/pull/112

**Pull request closed without merging**